src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -86,11 +74,11 @@ angular
             name = _.startCase _.trim name.toLowerCase().replace(/\s*projected\s*/, ' ')
           "<strong>#{date}</strong><br>#{name}: #{amount}"
 
-    todayMarker: ->
-      return {} unless @options.showToday
+    todayMarker: (settings) ->
```

Comment:
`todayMarker: (showToday, markerColor = 'rgba(0, 85, 255, 0.2)')` ?

Reply:
Yeah that works.